sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -15,15 +15,18 @@ namespace Azure.Core
     internal abstract class OperationInternalBase
     {
         private readonly ClientDiagnostics _diagnostics;
-        private readonly string _updateStatusScopeName;
         private readonly IReadOnlyDictionary<string, string>? _scopeAttributes;
         private readonly DelayStrategy? _fallbackStrategy;
         private readonly AsyncLockWithValue<Response> _responseLock;
 
+        protected readonly string _updateStatusScopeName;
```

Comment:
Why is this protected and not private?

Reply:
I changed the field to `_operationTypeName` but still protected because its sub class `OperationInternal<T>` needs it.

Reply:
I think that is ok for now, we can revisit this when we refactor these types as part of MQ.  @AlexanderSher FYI